test/support/workarea/global_e_support.rb

```diff
@@ -16,3 +16,8 @@ def reset_global_e_config
     end
   end
 end
+
+# The factory data uses these exchanges rates, so because we exchange them
+# for metrics, having example rates is necessary.
+Money.default_bank.add_rate('EUR', 'USD', 1.21)
+Money.default_bank.add_rate('USD', 'EUR', 0.83)
```

Comment:
I'm confused why these are needed, because we have the order price in both the localized currency and the merchants base currency and should be using the prices recorded in global e not converted through the money gem.

Reply:
When the order gets placed, metrics get logged here: https://github.com/workarea-commerce/workarea/blob/v3.5-stable/core/app/models/workarea/metrics/user.rb#L48-L49

And so it was failing because factory global e orders were exchanging. Are you saying it should be pulling that data from other fields to get the global e exchanged values? If so, what fields?

Reply:
(note there are many places that exchange happens, it might actually be here, I forget: https://github.com/workarea-commerce/workarea/blob/v3.5-stable/core/app/models/workarea/metrics/by_day.rb#L28)

Reply:
@eric-pigeon ping?

Reply:
Depends on what data you are looking for.  There are fields on the order item https://github.com/workarea-commerce/workarea-global-e/blob/master/app/models/workarea/order/item.decorator#L4 which also embed `international_price_adjustments`.  Fields on the order https://github.com/workarea-commerce/workarea-global-e/blob/master/app/models/workarea/order.decorator#L15.  Order Level Discounts are stored in a price adjustment international_discount_adjustments because GlobalE has its own method of prorating discounts which they don't expose in their data model.

Reply:
Ok, maybe there's a misunderstanding. The metrics code does an exchange to the default currency so metrics are handled consistently in the default currency for the app. It's currently using the out-of-the-box fields obviously, like `Order#total_price`. You're saying we have the order price in the localized currency and the default currency, so: which fields are the ones with the default currencies that we should be using in metrics? I saw the fields you're talking about, but it wasn't clear to me how they would be used.